.github/workflows/stale.yml

```diff
@@ -0,0 +1,25 @@
+# https://github.com/marketplace/actions/close-stale-issues
+name: "Stale issues"
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+permissions:
+  issues: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5
+        with:
+          # issue configuration
```

Comment:
Do you think we'd also want  [close-issue-message](https://github.com/marketplace/actions/close-stale-issues#close-issue-message)

```suggestion
          # issue configuration
         close-issue-message: "This issue is closed due to inactivity. Feel free to reopen the issue if you have more follow-ups!"
```

Reply:
sounds nice to me <img src="https://slackmojis.com/emojis/10804-meow_love/image/1643515051/meow_love.png" data-canonical-src="https://slackmojis.com/emojis/10804-meow_love/image/1643515051/meow_love.png" width="15" height="15" />